src/CommandingOfficers/Meridian.java

```diff
@@ -0,0 +1,375 @@
+package CommandingOfficers;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.Modifiers.CODamageModifier;
+import CommandingOfficers.Modifiers.CODefenseModifier;
+import Engine.GameAction;
+import Engine.GameActionSet;
+import Engine.Path;
+import Engine.UnitActionType;
+import Engine.Utils;
+import Engine.XYCoord;
+import Engine.Combat.BattleInstance.BattleParams;
+import Engine.GameEvents.GameEvent;
+import Engine.GameEvents.GameEventListener;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameInput.GameInputState;
+import Engine.GameInput.GameInputState.StateData;
+import Terrain.GameMap;
+import Terrain.Location;
+import Terrain.MapMaster;
+import UI.MapView;
+import UI.Art.Animation.GameAnimation;
+import Units.Unit;
+import Units.UnitModel;
+import Units.Weapons.Weapon;
+import Units.Weapons.WeaponModel;
+
+/**
+ * Commander Meridian focuses on a unique brand of flexibility; she transforms her tanks and artillery into each other 
+ */
+public class Meridian extends Commander
+{
+  private static final CommanderInfo coInfo = new instantiator();
+  private static class instantiator extends CommanderInfo
+  {
+    public instantiator()
+    {
+      super("Meridian");
+      infoPages.add(new InfoPage(
+          "While other commanders may focus on fine-grain tactics, Meridian has taken a different line: her basic tanks and artillery are built from the same parts, and can be re-configured on the field at no cost.\n" + 
+          "This gives her forces extreme adaptability, and her powers hone this to a razor edge."));
+      infoPages.add(new InfoPage(
+          "Passive:\n" + 
+          "- Tanks and artillery both cost 6500\n" +
+          "- Tanks and artillery may transform into each other at no cost"));
+      infoPages.add(new InfoPage(
+          "Change and Flow ("+ChangeAndFlow.COST+"):\n" +
+          "Gives an attack and defense boost of "+ChangeAndFlow.BASIC_BUFF+"%\n" +
+          "Any unit that has transformed this turn is refreshed."));
+      infoPages.add(new InfoPage(
+          "Vehicular Charge ("+VehicularCharge.COST+"):\n" +
+          "Gives an attack and defense boost of "+VehicularCharge.BASIC_BUFF+"%\n" +
+          "Refreshes all land vehicles, but refreshed units suffer an attack and defense penalty of "+POST_REFRESH_STAT_ADJUSTMENT+"%\n"));
+    }
+    @Override
+    public Commander create()
+    {
+      return new Meridian();
+    }
+  }
+
+  /** A list of all the units I've refreshed and need to nerf. */
+  private ArrayList<Unit> justTransformed = new ArrayList<Unit>();
+  private ArrayList<Unit> toBeNerfed = new ArrayList<Unit>();
+  private static final int POST_REFRESH_STAT_ADJUSTMENT = -25;
```

Comment:
This feels like it should be a public member of `VehicularCharge`. Also, a 25% firepower nerf (well, effectively 15%) feels kinda bad on a 7-cost ability.

Reply:
Hmm. I get what you're saying, but it's a substantial chunk of Eagle's Lightning Strike utility and she's got a better D2D.

Could drop it to 6, though, I guess. It's a substantial stat difference and she doesn't refresh air/sea.